src/clj_pgp/message.clj

```diff
@@ -157,6 +158,22 @@
   (readable
     [packet opts]
     ;; PGPLiteralData is always readable
+    packet)
+
+
+  PGPMarker
+
+  (reduce-message
+    [packet opts rf acc]
+    ;; Such a packet MUST be ignored when received. It may be placed at the
+    ;; beginning of a message that uses features not available in PGP 2.6.x
+    ;; in order to cause that version to report that newer software is
+    ;; necessary to process the message.
+    [])
```

Comment:
This should return `acc`, since we don't know ahead of time what kind of value the caller will have passed in.

Reply:
Good catch.